### ISSUE TITLE
Swiss curves page text update

### ISSUE THREAD

**Issue by mcarrara-bioinfo** [GenSpectrum/cov-spectrum-website#800]
Up until recently, Swiss wastewater curves have been published both on the cov-spectrum website and on a dedicated ETHZ CBG webpage (https://bsse.ethz.ch/cbg/research/computational-virology/sarscov2-variants-wastewater-surveillance.html). After discussing with all parties involved it was decided to deprecate the CBG webpage and publish only on cov-spectrum.
Important text regarding methods, people involved and links should be included in cov-spectrum from the CBG page to finalise the transition.

Details on the final content that reached consensus can be found on a [dedicated Google document](https://docs.google.com/document/d/1149ReJz4j_h1UjLitJUHqYYC2mzOxX6uzJrj526MzYk)

To summarise the changes listed in the linked Google document:
- expand the introduction before the curves with details on the challenges of wastewater detection
- Include a `data sources` section
- Include the links to the video presentation of the project
- Include an `acknowledgments` section
- Include a `funding` section
- Include a `contacts` section

The hyperlinks to include are available either embedded in the text or as comments in the document.
